### ISSUE TITLE
Configurable sound notification channel

### ISSUE THREAD

**Issue by Macaddict42** [Vysci/LFG-Bulletin-Board#54]
By default, PlaySound() only plays to the sound effects channel. It would be great if there was a way to configure which channel its played on, or at least default to "master" channel.

**Comment by chy8803** [comment]
Yeah. Agree with this. The small DING sound volume is too low for me.  And I am wondering if we can have an option to choose a different alert sound source. 
Thanks

**Comment by akadevvy** [comment]
Please look into this, currently it's not audible. Would be nice to have this while leveling 1-60 vs. leaving it open all the time looking for vanilla dungeons.
